test/game_reader_test.rb

```diff
@@ -1,9 +1,22 @@
 gem 'minitest'
-require_relative '../lib/game_reader'
+require './lib/game_reader'
 require 'minitest/autorun'
 require 'minitest/pride'
 require 'pry'
 
 class GameReaderTest < Minitest::Test
 
+  def test_letters_have_scores
+    new_game = GameReader.new
+    expected = 1
+    result = new_game.score("A")
+    assert_equal expected, result
+  end
+
+  def test_can_check_words
+    new_game = GameReader.new
+    expected = 10
+    result = new_game.score("heyo")
+    assert_equal expected, result
+  end
```

Comment:
Having the support methods mentioned above would allow you to include some unit tests here.

Would also expect tests for `nil` and `""`.